### DISCUSSION TITLE
MOTION-BEAT-Frontend Discussion with CTB

### DISCUSSION THREAD

**Discussion by AnyTHIW** [motionbeat/MOTION-BEAT-Frontend#39] [Announcements]
mention: @CamelIsTheBestConvention 

주제1: code 생산성 향상을 위한 정기적인 소통시간 3시 7시?

주제2: #17 제목이 부적절. 모달이 아닌데 이름수정?. 완료상황? 그리고 이슈를 끝내고 따른 것들 분화시키기??.

*memo vscode extension git 변경기록으로 확인가능!

**Comment by moonmisae118** [comment]
주제1 Ans : 3시 7시 확인
주제2 Ans : 방만들기 모달을 컴포넌트로 변경, 이슈 이름은 추후 수정 예정
